crates/dids/Cargo.toml

```diff
@@ -7,18 +7,24 @@ repository.workspace = true
 license-file.workspace = true
 
 [dependencies]
+base64 = { workspace = true }
+byteorder = "1.5.0"
+chrono = { workspace = true }
 crypto = { path = "../crypto" }
 did-jwk = "0.1.1"
 did-web = "0.2.2"
 jwk = { path = "../jwk" }
 keys = { path = "../keys" }
+pkarr = "1.1.5"
 regex = "1.10.4"
+reqwest = { version = "0.11", features = ["blocking"] }
```

Comment:
if I'm interpreting [this](https://docs.rs/reqwest/0.12.4/reqwest/#wasm) correctly, `blocking` isn't supported in wasm

> The Client implementation automatically switches to the WASM one when the target_arch is wasm32, the usage is basically the same as the async api. Some of the features are disabled in wasm : [tls](https://docs.rs/reqwest/0.12.4/reqwest/tls/index.html), [cookie](https://docs.rs/reqwest/0.12.4/reqwest/cookie/index.html), [blocking](https://docs.rs/reqwest/0.12.4/reqwest/blocking/index.html).

That's still okay with me if we want to stick with it in this PR, but @diehuxx if you agree with my interpretation of the matter, could you open a ticket on the backlog to explore implications & alternatives.

Reply:
(wasm is a non-priority right now but I want to earmark it because I suspect wasm will be a priority at some point)